File: code.rb
ITERATIONS = 10000000

start_time = Time.now

sum = 0
ITERATIONS.times do |i|
  100.times do |j|
    sum += i * j
  end
end

end_time = Time.now
duration = end_time - start_time

puts "Ruby Result: #{sum}"
puts "Ruby Time taken: #{duration} seconds"

File: code.py
import time

ITERATIONS = 1000000

start_time = time.time()

sum = 0
for i in range(ITERATIONS):
    for j in range(100):
        sum += i * j

end_time = time.time()
duration = end_time - start_time

print(f"Python Result: {sum}")
print(f"Python Time taken: {duration} seconds")
